src/components/search/search-bar.tsx

```diff
@@ -145,31 +154,64 @@ export const SearchBar: FunctionComponent<SearchBarProps> = ({ inputRef }) => {
         },
         [
             selectedDirectory?.elementUuid,
-            elementsFound,
             handleDispatchDirectory,
             updateMapData,
             snackError,
             dispatch,
+            elementsFound,
         ]
     );
 
     return (
         <ElementSearchInput
             sx={{ width: '50%', marginLeft: '14%' }}
             size="small"
-            elementsFound={elementsFound}
-            getOptionLabel={(element) => element.name}
-            isOptionEqualToValue={(element1, element2) =>
-                element1.id === element2.id
-            }
+            elementsFound={elementsFound as ElementAttributesES[]}
+            getOptionLabel={(element: ElementAttributesES) => element.name}
+            isOptionEqualToValue={(
+                element1: ElementAttributesES,
+                element2: ElementAttributesES
+            ) => element1.id === element2.id}
             onSearchTermChange={updateSearchTerm}
             onSelectionChange={handleMatchingElement}
             renderElement={renderOptionItem}
             searchTerm={searchTerm}
             loading={isLoading}
-            renderInput={(value, params) => (
+            renderInput={(_value: any, params) => (
                 <SearchBarRenderInput inputRef={inputRef} {...params} />
             )}
+            PaperComponent={({
+                children,
+                ...other
+            }: {
+                children: React.ReactNode;
+            }) => {
+                return (
+                    <Paper {...other}>
+                        <Typography
+                            variant="body1"
+                            style={{
+                                color: theme.palette.info.main,
```

Comment:
Proposition: Move the style definition to the top of the class.

Reply:
i don't know it is possible to do in this case, because i am using the theme that comes from a custum hook

Reply:
fixed by Kevin